rasa/nlu/model.py

```diff
@@ -385,3 +388,11 @@ def parse(
         output = self.default_output_attributes()
         output.update(message.as_dict(only_output_properties=only_output_properties))
         return output
+
+    def parse_message(self, message: Message, attribute: Text = TEXT) -> Message:
+        for component in self.pipeline:
+            if isinstance(component, Featurizer) or isinstance(
+                component, tokenizer.Tokenizer
+            ):
+                component.process(message, attribute, **self.context)
+        return message
```

Comment:
this is unused according to Pycharm. Can we remove it?

Reply:
Pycharm is right but it is used in core featurization. I tried to add all the NLU changes here.

Reply:
Ok 👍 Sure it shouldn't be a method of `NaturalLanguageInterpreter` then?

Reply:
It calls this `Interpreter` inside it; that's why this function is needed here. I'll add a docstring though